repos/system_upgrade/el7toel8/actors/tcpwrappersconfigread/libraries/library.py

```diff
@@ -0,0 +1,67 @@
+import errno
+
+from leapp.libraries.stdlib import api
+from leapp.models import DaemonList, TcpWrappersFacts
+
+CONFIG_FILES = ['/etc/hosts.allow', '/etc/hosts.deny']
+
+
+def read_file(path):
+    with open(path, 'r') as f:
+        return f.read()
+
+
+def get_lines(content):
```

Comment:
Wondering if this will do the trick

```
re.sub(r'//[\S\t\n]*\n', '', content)
return content.split('\n')
```

Reply:
You mean this, right? Good thinking, I'll change it.
```
content = re.sub(r'\\\n', '', content)
return content.split('\n')
```